contracts/swappa/PairSavingsCELO.sol

```diff
@@ -33,5 +35,22 @@ contract PairSavingsCELO is ISwappaPairV1 {
     }
 	}
 
+	function getOutputAmount(
+		address input,
+		uint amountIn,
+		bytes calldata data
+	) external view override returns (uint amountOut) {
+		address savingsCELOAddr = parseData(data);
+		if (input == 0x471EcE3750Da237f93B8E339c536989b8978a438) {
+			// CELO
+			return ISavingsCELO(savingsCELOAddr).celoToSavings(amountIn);
+		} else if (input == 0x2879BFD5e7c4EF331384E908aaA3Bd3014b703fA) {
```

Comment:
this isn't correct since there is no way to convert from sCELO -> CELO in swappa because of unlocking period. 
Also hardcoding addresses isn't a good idea because it will cause issues in testing and testnets. Probably better to just revert to code that you had before.